app/models/setting/general.rb

```diff
@@ -22,7 +22,8 @@ def self.load_defaults
         self.set('use_gravatar', N_("Foreman will use gravatar to display user icons"), false, N_('Use Gravatar')),
         self.set('db_pending_migration', N_("Should the `foreman-rake db:migrate` be executed on the next run of the installer modules?"), true, N_('DB pending migration')),
         self.set('db_pending_seed', N_("Should the `foreman-rake db:seed` be executed on the next run of the installer modules?"), true, N_('DB pending seed')),
-        self.set('proxy_request_timeout', N_("Max timeout for REST client requests to smart-proxy"), 60, N_('Proxy request timeout'))
+        self.set('proxy_request_timeout', N_("Max timeout for REST client requests to smart-proxy"), 60, N_('Proxy request timeout')),
+        self.set('released_versions', N_("Foreman tries to find out about released versions"), false, N_('Find Foreman releases'))
```

Comment:
also, I would consider changing this to a more generic statement, it is true that currently it only compare version information, but I could see how other plugins and additional information could be used, and would prefer not to keep creating additional settings for each call home feature.